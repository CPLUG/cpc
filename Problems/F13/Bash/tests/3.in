10
123
12398sd
1 20398
24302
3209jva
loldongs.txt
3loldongs
3
lol
.txt
