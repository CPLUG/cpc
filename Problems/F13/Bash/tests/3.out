{.txt,1{ 20398,23{,98sd}},24302,3{,209jva,loldongs},lol{,dongs.txt}}
